spark/src/test/java/org/apache/iceberg/spark/source/TestDataSourceOptions.java

```diff
@@ -196,6 +196,6 @@ public void testSplitOptionsOverridesTableProperties() throws IOException {
         .option("split-size", String.valueOf(562L)) // 562 bytes is the size of SimpleRecord(1,"a")
         .load(tableLocation);
 
-    Assert.assertEquals("Spark partitions should match", 2, resultDf.javaRDD().getNumPartitions());
+    Assert.assertEquals("Spark partitions should match", 4, resultDf.javaRDD().getNumPartitions());
```

Comment:
This change is suspicious. Why did the number of partitions increase?

Reply:
I've also mentioned this on the devlist, the parquet file size grew a bit bigger. Probably the column indexes, CRC checksum, removing the statistics from the page headers and maybe other changes that impact file size.

Reply:
I've written a single record, and updated the size to 611:
```
$ ls -la /var/folders/km/xypq2kxs4ys3dt6bwtd4fbj00000gn/T/junit8105143976887819489/iceberg-table/data/
total 16
drwxr-xr-x  4 fokkodriesprong  staff  128 11 dec 21:51 .
drwxr-xr-x  4 fokkodriesprong  staff  128 11 dec 21:51 ..
-rw-r--r--  1 fokkodriesprong  staff   16 11 dec 21:51 .00000-0-44b7bf06-961b-4f36-9350-8711db220a8e-00000.parquet.crc
-rw-r--r--  1 fokkodriesprong  staff  611 11 dec 21:51 00000-0-44b7bf06-961b-4f36-9350-8711db220a8e-00000.parquet
```